### PR TITLE
use the latest stable version of Nushell in the CI

### PR DESCRIPTION
**Pull request by amtoine**
might supersede:
- https://github.com/amtoine/nu-git-manager/pull/36

related to
- https://github.com/hustcer/setup-nu/issues/58

## description
this is an experiment to see if that could work without pulling Nightly.
there is an [open issue in the GitHub action](https://github.com/hustcer/setup-nu/issues/58) for that :+1:

### CODE REVIEW

**Comment by amtoine** [comment]
ehe it works :yum: 

@melMass 
do you think this is better than #36? :innocent:

**Comment by melMass** [comment]
Awesome! I'm just heading out but I'll have a look in 2h

**Comment by amtoine** [comment]
@melMass 
fwiw, https://github.com/amtoine/nu-git-manager/pull/38 is an example of PR that will need to wait for the next release to land because the changes are in 0.86.1 :+1:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,54 +26,17 @@ jobs:
...
-
-          curl -fLo $name $url
-
-          if [ "${{ env.EXT }}" = "tar.gz" ]; then
-            tar xvf $name --directory /tmp
-          elif [ "${{ env.EXT }}" = "zip" ]; then
-            unzip $name -d "/tmp/$name"
-          fi
-          cp "/tmp/$name/${{ env.NU_BIN }}" "$HOME/${{ env.NU_BIN }}"
+      - uses: hustcer/setup-nu@v3.6
```
Comment (hustcer) [inline .github/workflows/ci.yml:29]: - uses: hustcer/setup-nu@v3.7
        with:
          version: nightly

Comment (amtoine) [reply .github/workflows/ci.yml:29]: amazing :star_struck: 

i think we'll be focusing on the last stable release for now but it's so great to have that in mind if we need to :pray:

**Comment by amtoine** [comment]
@melMass 
as you can see in https://github.com/nushell/nupm/pull/40, using the *nightly* builds of Nushell after this PR would be trivial :ok_hand: 

i think we should close #36 and go with the GitHub action, it's much much simpler :relieved:

**Comment by amtoine** [comment]
let's land this, we can always use the new nightly GitHub action to target the latest builds :wink:

**Pull request merged**